### ISSUE TITLE
Suggestion: janitor package

### ISSUE THREAD

**Issue by smach** [kbroman/hipsteR#7]
adorn_totals() to add a totals column, totals row, or both, is very nice. Some other functions worth knowing, such as tabyl() 
https://cran.r-project.org/web/packages/janitor/vignettes/introduction.html

**Comment by kbroman** [comment]
Thanks! I added it to the section on Hadley's packages, which probably should just be re-titled Tidyverse.

**Comment by smach** [comment]
janitor isn't a Hadley/Tidyverse package, it was written by Sam Firke. Probably ought to be in another section.

**Comment by kbroman** [comment]
Yeah, okay. I'll just omit it for now. Not sure where to fit it in.
